.devops/pagopa-code-review-pipelines.yml

```diff
@@ -34,7 +34,7 @@ resources:
     - repository: checkout-be-mock
       type: github
       name: pagopa/pagopa-checkout-be-mock
-      ref: master
+      ref: CHK-925-added-fault-cases-for-verify-method
```

Comment:
I wait for the branch merge in the pagopa-checkout-be-mock project to change the branch reference